RecoLocalCalo/HGCalRecAlgos/interface/HGCalUncalibRecHitRecAbsAlgo.h

```diff
@@ -0,0 +1,27 @@
+#ifndef RecoLocalCalo_HGCalRecAlgos_HGCalUncalibRecHitRecAbsAlgo_HH
+#define RecoLocalCalo_HGCalRecAlgos_HGCalUncalibRecHitRecAbsAlgo_HH
+
+/** \class HGCalUncalibRecHitRecAbsAlgo
+  *  Template used by Ecal to compute amplitude, pedestal, time jitter, chi2 of a pulse
+  *  using a weights method
+  *
+  *  \author
+  */
+
+#include "Math/SVector.h"
+#include "Math/SMatrix.h"
```

Comment:
Math headers are not needed in this header file based on what's used/declared here. Please cleanup or move to .cc